packages/adapters-library/src/adapters/aave-v2/common/aaveBasePoolAdapter.ts

```diff
@@ -181,7 +262,7 @@ export abstract class AaveBasePoolAdapter
     reserveData: Awaited<ReturnType<ProtocolDataProvider['getReserveData']>>,
   ): bigint
 
-  private async fetchPoolMetadata(protocolTokenAddress: string) {
+  async fetchPoolMetadata(protocolTokenAddress: string) {
```

Comment:
Why is this no longer private? I cannot find any instance of it being called outside of the class context.